pkg/sdkserver/sdk_test.go

```diff
@@ -75,13 +79,33 @@ func TestConvert(t *testing.T) {
 		}
 	}
 
-	sdkGs := convert(fixture)
-	eq(t, fixture, sdkGs)
-	assert.Zero(t, sdkGs.ObjectMeta.DeletionTimestamp)
+	t.Run(string(runtime.FeaturePlayerTracking)+" disabled", func(t *testing.T) {
+		assert.NoError(t, runtime.ParseFeatures(""))
+
+		gs := fixture.DeepCopy()
+
+		sdkGs := convert(gs)
+		eq(t, fixture, sdkGs)
+		assert.Zero(t, sdkGs.ObjectMeta.DeletionTimestamp)
+		assert.Nil(t, sdkGs.Status.Players)
+
+		now := metav1.Now()
+		gs.DeletionTimestamp = &now
+		sdkGs = convert(gs)
```

Comment:
In this test you convert twice and in the test with the feature enabled you only do it once. Why the difference?

Reply:
The second convert was to test if the DeletionTimestamp was converted correctly. Once we had had tested that once, there didn't seem to be any reason to repeat the test.

The second block could be moved to it's own `t.Run()` block. That would probably read a little nicer too. WDYT?

Reply:
I think that is much clearer. Thanks.